tests/test_WavefrontData.py

```diff
@@ -0,0 +1 @@
+from SimExLite.WavefrontData import WavefrontData 
```

Comment:
pep8

Reply:
This is concerned by https://github.com/PaNOSC-ViNYL/SimEx-Lite/issues/16